src/octoprint/plugins/classicwebcam/static/less/classicwebcam.less

```diff
@@ -0,0 +1,183 @@
+#webcam_video_container {
```

Comment:
Am I right in assuming that this is extracted from `src/static/less/octoprint.less`?

Reply:
Copied in CSS, this is autogenerated from it